### PR TITLE
question card

### PR DESCRIPTION
**Pull request by alicie232**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,25 +5,61 @@
...
-        <button>Show Answer/Hide Answer</button>
-        <ul>
-          <li>Tag</li>
-          <li>Tag</li>
-          <li>Tag</li>
-          <li>Tag</li>
-        </ul>
+        <article class="question__card">
+          <h2 class="question__card--title">Question:</h2>
+          <!-- Java Script für output von create.html fehlt noch-->
```
Comment (mbosselmann) [inline index.html:22]: I don't think this comment is necessary to understand your code.

File: index.html [hunk 2]
```diff
@@ -5,25 +5,61 @@
...
+          <p class="question__card--question">
+            Hier steht eine Frage<br />
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolore hic
+            possimus eaque nam in perspiciatis laudantium doloribus enim.
+          </p>
+          <button class="question__card--button">
+            Show Answer
+            <!-- /Hide Answer -->
```
Comment (mbosselmann) [inline index.html:30]: Same here

File: index.html [hunk 3]
```diff
@@ -5,25 +5,61 @@
...
+          </button>
+          <div class="question__card--all-tags">
+            <div class="question__card--tag">Tag</div>
+            <div class="question__card--tag">Tag</div>
+            <div class="question__card--tag">Tag</div>
+            <div class="question__card--tag">Tag</div>
+          </div>
+          <a class="question__card--bookmark" href="bookmark.html" alt="Home">
+            <i class="fa-solid fa-bookmark"></i>
+          </a>
```
Comment (mbosselmann) [inline index.html:44]: This one is more of a question: I'm not sure if it is user-friendly that every time you click on a bookmark will guide the user to the bookmark page. I would suggest you to work with a button and let the user decide for himself or herself when he or she wants to see the bookmark page. If you want to work with a button, then the value of the `alt` tag should not be `Home`.

File: style/components/home.css [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  position: relative;
+  background-color: var(--background-color--secondary);
+  width: 300px;
+  margin: 30px;
+  padding: 15px;
+  gap: 15px;
+  border-radius: 20px;
+}
+
+.question__card--title {
```
Comment (mbosselmann) [inline style/components/home.css]: According to BEM the naming would be `question-card__title` as `--` means modifier and `__` means a nested element. [See examples for naming BEM.](http://getbem.com/naming/) 
Please check for all your other class names too.

**Review by mbosselmann — COMMENTED** [review]
🚀 Again I left some remarks for you to consider before merging. :)

File: style/components/home.css (hunk 4 above)
Comment (alicie232) [reply style/components/home.css]: Thank you!!! Have long thought about the names, but your objection sounds very logical to me and helps me a lot.

**Pull request merged**